### PR TITLE
WebAPI: Update constructor pages (part 6)

### PR DESCRIPTION
**Pull request by OnkarRuikar**
### Summary

Adds empty `### Parameters` section as per the guideline: 
https://developer.mozilla.org/en-US/docs/MDN/Structures/Syntax_sections

#### Metadata
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/focusevent/focusevent/index.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ new FocusEvent(typeArg);
 new FocusEvent(typeArg, focusEventInit);
```
Comment (teoli2003) [inline files/en-us/web/api/focusevent/focusevent/index.md:23]:
```suggestion
new FocusEvent(typeArg, focusEventInit)
```

(and also on the line before)

**Review by teoli2003 — CHANGES_REQUESTED** [review]

**Comment by OnkarRuikar** [comment]
done!

**Review by teoli2003 — APPROVED** [review]
Thanks a lot!

**Comment by teoli2003** [comment]
Woopsie there are a couple of conflicts here.

**Comment by OnkarRuikar** [comment]
done!

**Pull request merged**